### PR TITLE
Welcome panel & discard option workflow

### PR DESCRIPTION
**Pull request by mtrive**
**Problem**
Project Auditor supports several modules: code, settings, shaders, etc... Each reporting their own issues. Project Auditor always executes all of them but this is not ideal as the user might only be interested in a specific module (or a subset of modules).

**Solution**
This PR introduces a Modules selection in the "welcome" page.

modules-selection

In addition, it introduces a "discard" button to the analysis toolbar which allows the user to return to the initial state in order to start a new analysis.

discard

### CODE REVIEW

**Review by unitystevem — APPROVED** [review]
Approving on the grounds that it works as advertised, but I think there need to be future PRs which improve the workflow. If categories are not selected in the "Welcome" view, they should still appear in the "Summary" view but with a button that says "Analyze" rather than a link to the corresponding View. Clicking Analyze would perform that module's analysis and add it to the window (without deleting the analysis results from other modules).

Basically, having to delete all the code analysis or build reports to get back to the Welcome screen because you didn't select Project Issues first time around and now want to see them feels pretty clunky.

**Comment by mtrive** [comment]
> Basically, having to delete all the code analysis or build reports to get back to the Welcome screen because you didn't select Project Issues first time around and now want to see them feels pretty clunky.

I have to admit, I had the same feeling.
